airbyte-cdk/java/airbyte-cdk/datastore-postgres/src/main/kotlin/io/airbyte/cdk/db/PostgresUtils.kt

```diff
@@ -18,7 +18,11 @@ object PostgresUtils {
         val jsonNodes =
             database.bufferedResultSetQuery(
                 { conn: Connection ->
-                    conn.createStatement().executeQuery("select * from pg_current_wal_lsn()")
+                    conn
+                        .createStatement()
+                        .executeQuery(
+                            "SELECT CASE WHEN pg_is_in_recovery() THEN pg_last_wal_receive_lsn() ELSE pg_current_wal_lsn() END AS pg_current_wal_lsn;"
```

Comment:
This change should also be carried over to `getLsn` in `PostgresUtils` in the CDK. Please do this in a separate PR because CDK releases are annoying to manage otherwise.

Reply:
Hello @postamar, so a new version of Airbyte CDK needs to be published. Is this correct? 
Shall I do this in this PR and open an other where I use the new version of CDK in Postgres Source where I bump its DockerTag?

Reply:
@postamar could you please clarify the requested change here because Im not sure to get it fully 🙏

Reply:
@theyueli I think that was the initial change (bumping version in `airbyte-cdk` instead of source-postgres DockerTag. Could you please confirm that I need to revert to the initial change? Thanks 🙏

Reply:
@assaadhjb I spoke too fast... yes, your original change was correct. I thought `postgresutils.kt` is part of the connector not CDK. It is in the CDK. 

In this case, you just need to only bump up the CDK version as you did before. In addition, please change CDK's changelog `airbyte-cdk/java/airbyte-cdk/README.md`.  sorry for the confusion.

Reply:
@theyueli No problem! I'll restore the old changes so we upgrade CDK and use it in a separate PR. Thanks again for your reviews 🙏